### DISCUSSION TITLE
How to get the transformed asymmetric Eular angles?

### DISCUSSION THREAD

**Discussion by Dxc0000** [mtex-toolbox/mtex#2081] [Ask Anything]
Hello, everyone
Here is my problem: 
For a fcc sample grain boundary segment, here are grain boundary orientations ori1 and ori2 on the either side of this grain boundary, which are both demonstrated by Euler angles.
First, I want to reduce the calculated misorientation into Fundamental zone by symmetric operations. However, when I checked the operations in mtex, I only found how to plot it instead of acquisition of the transformed Euler angle data. I'd like to ask how to deal with this problem.
Second, after the misorientation is reduced into this fundamental zone, one orientation must be symmetrically changed to match the transformed misorientation. Also, is there any function can be used directly to achieve this goal? Or I have to write a loop code to check out which one fits?
Look forward to your reply, 
Thanks

**Comment by kilir** [comment]
Hi,

I guess this might help with your first question:

```Matlab
cs = crystalSymmetry('m-3m');
ori =orientation.rand(2,cs);

mori = inv(ori(1)) * ori(2)

% check whether inside or not
fR = fundamentalRegion(cs,cs);
fR.checkInside(mori)

% eventually project to the fR
mori = mori.project2FundamentalRegion;
```
With respect to your second question, I'm not entirely sure what you mean. Project the orientation to the fundamentalRegion?

Cheers,

Rüdiger

**Reply by Dxc0000** [reply]
Hi, Rüdiger,
Thank you so much for your reply. Can't tell how grateful I am.
As for my second question, what I mean is that after the misorientation is calculated and projected to the fundamentalRegion, how can I know what symmetry transformations are implemented to these two orientations to get this projected misorientation?
Like this:
mori = inv(ori(1))*ori(2)
mori_{project2FR} = inv(C1*ori(1))*(C2*ori(2))
How can I know what C1 and C2 are (C1 and C2 are symmetry operators)?
